test/plugins/adonis.spec.js

```diff
@@ -0,0 +1,346 @@
+'use strict'
+
+const axios = require('axios')
+const getPort = require('get-port')
+const agent = require('./agent')
+const plugin = require('../../src/plugins/adonis')
+const { setupResolver, Config, Logger } = require('@adonisjs/sink')
+const { ioc } = require('@adonisjs/fold')
+const path = require('path')
+
+const Server = require('@adonisjs/framework/src/Server')
```

Comment:
These should all be imported from versions.  See: https://github.com/signalfx/signalfx-nodejs-tracing/blob/master/test/plugins/ioredis.spec.js#L17 for an example.